### ISSUE TITLE
failed to load

### ISSUE THREAD

**Issue by gillyspy** [moyicat/inkdrop-code-fold#6]
![image](https://user-images.githubusercontent.com/1345313/197396216-43ad1252-d3ac-4468-b7a5-4fc4519cc853.png)

```
private/var/folders/t3/7y1tl95d5qb82c8n5d4rp4000000gn/T/AppTranslocation/E1CF2F47-EC64-4014-9CD3-146C75DDBCCB/d/Inkdrop.app/Contents/Resources/app.asar/node_modules/debug/src/common.js:113 app:error Notification: +0ms Failed to load the code-fold package {stack: 'TypeError: Cannot read properties of undefined (re…ents/Resources/app.asar/browser-main.js:1:482465)', detail: "Cannot read properties of undefined (reading 'app')", packageName: 'code-fold', dismissable: true}
debug @ /private/var/folders/t3/7y1tl95d5qb82c8n5d4rp4000000gn/T/AppTranslocation/E1CF2F47-EC64-4014-9CD3-146C75DDBCCB/d/Inkdrop.app/Contents/Resources/app.asar/node_modules/debug/src/common.js:113
(anonymous) @ browser-main.js:1
simpleDispatch @ <embedded>:98274
emit @ <embedded>:98467
addNotification @ browser-main.js:1
addFatalError @ browser-main.js:1
handleError @ browser-main.js:1
(anonymous) @ browser-main.js:1
measure @ browser-main.js:1
load @ browser-main.js:1
loadPackage @ browser-main.js:1
loadPackages @ browser-main.js:1
start @ browser-main.js:1
await in start (async)
26151 @ browser-main.js:1
n @ browser-main.js:1
84303 @ browser-main.js:1
n @ browser-main.js:1
(anonymous) @ browser-main.js:1
n.O @ browser-main.js:1
(anonymous) @ browser-main.js:1
(anonymous) @ browser-main.js:1
```

**Comment by gillyspy** [comment]
inkdrop v5.5.1

**Comment by craftzdog** [comment]
Fixed in code-fold@1.4.1

**Comment by gillyspy** [comment]
it loads now
